control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -229,6 +234,19 @@ func Setup(ctx context.Context, opts *operator.HostedClusterConfigOperatorConfig
 	if err := c.Watch(source.Kind[client.Object](opts.CPCluster.GetCache(), &hyperv1.HostedControlPlane{}, eventHandler())); err != nil {
 		return fmt.Errorf("failed to watch HostedControlPlane: %w", err)
 	}
+	// HCCO needs to watch for KubeletConfig ConfigMaps on the Control plane cluster (MNG cluster)
+	// and mirrors them to the hosted cluster so the operator on the hosted cluster
```

Comment:
which operator? can we clarify this is driven by NTO consumers and mention an example?

Reply:
Added. Thanks